app/src/main/java/org/simple/clinic/patient/OngoingNewPatientEntry.kt

```diff
@@ -113,6 +114,17 @@ data class OngoingNewPatientEntry(
   fun withZone(zone: String): OngoingNewPatientEntry =
       copy(address = addressOrBlank().copy(zone = zone))
 
+  fun withPatientPrefillInfo(patientProfileInfo: PatientPrefillInfo, identifier: Identifier): OngoingNewPatientEntry =
```

Comment:
Created a [story](https://app.clubhouse.io/simpledotorg/story/3639/resolve-coupling-between-scanid-and-instantsearch-packages?vc_group_by=day&ct_workflow=all) for this will resolve in another PR.